packml_msgs/msg/ItemizedStats.msg

```diff
@@ -0,0 +1,6 @@
+# This message contains itemized statics for PackML statistics reporting
+# Each item containds a unique
+
+int16 id                          # unique identifier (user defined)
+int32 count                       # number of occurances
```

Comment:
I hadn't noticed that.  I didn't use `int32` on purpose.  In fact, in this case, the `count` should always be positive.